src/object_detection.cpp

```diff
@@ -1,35 +1,39 @@
 #include <ros/ros.h>
-#include <sensor_msgs/Image.h>
-#include <object_detection_pkg/ObjDetected.h>
 #include <ros/console.h>
+#include <ros/package.h>
+#include <sensor_msgs/Image.h>
+#include <sensor_msgs/image_encodings.h>
 #include <geometry_msgs/Vector3.h>
-#include <fstream>
-#include <cstdlib>
 #include <std_msgs/String.h>
-#include "DetectedObject.h"
 #include <image_transport/image_transport.h>
-#include <cv_bridge/cv_bridge.h>
-#include <sensor_msgs/image_encodings.h>
-#include <opencv2/highgui/highgui.hpp>
+#include <object_detection_pkg/ObjDetected.h>
+#include <object_detection_pkg/DetectedObjsArray.h>
+#include <fstream>
+#include <cstdlib>
```

Comment:
It doesn't look like this header is being used either, just double-check your includes that they are being used and aren't outdated!

Reply:
I checked and fstream is needed but cstdlib is not. I removed the header reference.